models/cities.model.js

```diff
@@ -1,24 +1,30 @@
-import database from '../database.js'
+import { City } from '../db/models/models.js'
 
 export default {
   getCities: async () => {
-    const cities = await database.query('SELECT * FROM cities')
-    return cities.rows
+    const cities = await City.findAll()
+    return cities
```

Comment:
The cities object has a lot of information from ORM, returning only an array of cities.
Do that in this case and in the others model also